app/components/create_log_actions_component.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if display_actions? %>
```

Comment:
feel like I have to suggest a helper as there are nested ifs, just my personal limit (also occurs in app/views/logs/_create_for_org_actions.html.erb for example)